libraries/standard/http/include/http_client.h

```diff
@@ -73,6 +73,20 @@
  */
 #define HTTP_RESPONSE_CONNECTION_KEEP_ALIVE_FLAG    0x2U
 
+/**
+ * @brief Flag that represents End of File byte in the range specification of
+ * a Range Request.
+ * This flag should be used ONLY for 2 kinds of range specifications when
+ * creating the Range Request header through the #HTTPClient_AddRangeHeader
+ * function:
+ *  - When the requested range is all bytes from the starting range byte to
+ * the end of file.
+ *  - When the requested range is for the last N bytes of the file.
+ * In both cases, this value should be used for the "rangeEnd" parameter.
+ */
+#define HTTP_RANGE_REQUEST_END_OF_FILE              -1
```

Comment:
`0xFF` is a valid byte range end value for a customer to provide (for example, file bytes in the range [0-255], [255, EoF]).
Thus, a negative value is reserved for the macro as a customer should not pass a negative value otherwise to represent EoF